<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>voOpenTreeLoadDialog</class>
 <widget class="QDialog" name="voOpenTreeLoadDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>404</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Load Tree from OpenTreeOfLife Database</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="1">
    <widget class="QLineEdit" name="hostURL">
     <property name="toolTip">
      <string>Open Tree Of Life database URL</string>
     </property>
     <property name="text">
      <string>http://opentree-dev.bio.ku.edu:7474/db/data/ext/GoLS/graphdb/getDraftTreeForOttolID</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="databaseLabel">
     <property name="text">
      <string>Database:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="ottolID">
     <property name="text">
      <string>386191</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>maxDepth:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="collectionLabel">
     <property name="text">
      <string>ottolID:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="databaseName">
     <property name="text">
      <string>opentree</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="hostLabel">
     <property name="text">
      <string>Host:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QLineEdit" name="maxDepth">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>voOpenTreeLoadDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>voOpenTreeLoadDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>201</x>
     <y>88</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
